_source/_posts/2023-06-30-podcast-approaching-identity-automation.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: blog_post
+title: "June Podcast: Approaching Automation with Jeff Taylor"
+author: edunham
+by: advocate
+communities: [devops,security]
+description: "Listen and learn about the automation journey"
+tags: [podcast, automation, enterprise]
+image: blog/identity-developer-podcast/podcast.jpg
+type: awareness
+---
+
+# Approaching Automation
```

Comment:
```suggestion
# Approaching automation in identity
```

Reply:
actually I just realized this is a h1. this should be deleted (always start with h2) and we don't need a header here